sfa_dash/templates/head.html

```diff
@@ -4,7 +4,17 @@
 <meta name="author" content="">
 <link rel="icon" href="/static/images/favicon.ico">
 
-<title>Solar Forecast Arbiter</title>
+<title>
+{% if metadata is defined and 'name' in metadata %}
+{{ metadata['name'] }}
+{% elif metadata is defined and 'report_parameters' in metadata %}
+{{ metadata['report_parameters']['name'] }}
```

Comment:
We use `page title | Solar Forecast Arbiter` on the informational website. I could go either way on the `| Solar Forecast Arbiter` here on the dashboard.